scala-package/spark/src/test/scala/org/apache/mxnet/spark/MXNetGeneralSuite.scala

```diff
@@ -46,24 +46,26 @@ class MXNetGeneralSuite extends SharedSparkContext {
       "/dataset/mxnet-spark-test/train.txt" + " -P " + testDataDir + " -q") !
   }
 
-  override def beforeAll(): Unit = {
-    val tempDirFile = Files.createTempDirectory(s"mxnet-spark-test-${System.currentTimeMillis()}").
-      toFile
-    testDataDir = tempDirFile.getPath
-    tempDirFile.deleteOnExit()
-    downloadTestData()
-  }
-
+//  override def beforeAll(): Unit = {
+//  val tempDirFile = Files.createTempDirectory(s"mxnet-spark-test-${System.currentTimeMillis()}").
+//      toFile
+//    testDataDir = tempDirFile.getPath
+//    tempDirFile.deleteOnExit()
+//    downloadTestData()
+//  }
 
-  test("run spark with MLP") {
-    val trainData = parseRawData(sc, s"$testDataDir/train.txt")
-    val model = buildMlp().fit(trainData)
-    assert(model != null)
-  }
+  test("Dummy test on Spark") {
 
-  test("run spark with LeNet") {
-    val trainData = parseRawData(sc, s"$testDataDir/train.txt")
-    val model = buildLeNet().fit(trainData)
-    assert(model != null)
   }
+//  test("run spark with MLP") {
```

Comment:
No need to comment we can just remove, but not blocking it as this will have to go through another round of CI.